README.md

```diff
@@ -542,6 +542,48 @@ showCard(Pair('X', '♠'));
 // ! TypeError: ‘showCard’ expected a value of type (Pair Rank Suit) as its first argument; received Pair("X", "♠")
 ```
 
+#### `EnumType`
+
+`EnumType` is used to construct [enumerated types][5].
+
+To define an enumerated type one must provide:
+
+  - an array of values with distinct [`R.toString`][6] representations.
+
+```haskell
+EnumType :: [Any] -> Type
+```
+
+For example:
+
+```javascript
+//    TimeUnit :: Type
+const TimeUnit = $.EnumType(['milliseconds', 'seconds', 'minutes', 'hours']);
+
+const def = $.create($.env.concat([TimeUnit, $.ValidDate, $.ValidNumber]));
+
+//    convertTo :: TimeUnit -> ValidDate -> ValidNumber
+const convertTo =
+def('convertTo',
+    {},
+    [TimeUnit, $.ValidDate, $.ValidNumber],
+    function recur(unit, date) {
+      switch (unit) {
+        case 'milliseconds': return date.valueOf();
+        case 'seconds':      return recur('milliseconds', date) / 1000;
+        case 'minutes':      return recur('seconds', date) / 60;
+        case 'hours':        return recur('minutes', date) / 60;
+      }
+    });
+    
+convertTo('seconds', new Date(1000));
+// => 1
+
+convertTo('days', new Date(1));
```

Comment:
Let's use `new Date(1000)` as the second argument here as well, to draw the reader's attention to the first argument.